src/components/set/index.js

```diff
@@ -0,0 +1,96 @@
+import { useParams } from "react-router-dom";
+import { useEffect, useState } from 'react';
+import { Container, Row, Table } from 'react-bootstrap';
+import { setEV } from '../calculator';
+import { getSetInfo } from '../../api';
+import DataTable from "./table";
+
+function Set() {
+    let { setCode } = useParams();
+    const [packValue, setPackValue] = useState(null);
+    const [boxValue, setBoxValue] = useState(null);
+    const [filteredPackValue, setFilteredPackValue] = useState(null);
+    const [filteredBoxValue, setFilteredBoxValue] = useState(null);
+    const [totalSetData, setTotalSetData] = useState(null);
+    const [valueSetData, setValueSetData] = useState(null);
+    const [setName, setSetName] = useState(null);
+    const [cardCount, setCardCount] = useState(null);
+
+
+    const getSetData = async () => {
+        const { packValue, boxValue, filteredPackValue, filteredBoxValue, totalSetData, valueSetData } = await setEV(setCode);
+        if (packValue || boxValue || filteredPackValue || filteredBoxValue || totalSetData || valueSetData) {
+            setPackValue(packValue);
+            setBoxValue(boxValue);
+            setFilteredPackValue(filteredPackValue);
+            setFilteredBoxValue(filteredBoxValue);
+            setTotalSetData(totalSetData);
+            setValueSetData(valueSetData);
+        }
+        let { name, card_count } = await getSetInfo(setCode);
```

Comment:
Are setInfo and setEV doing the same fetch? If so you may be able to consolidate them into one return. Maybe name and card_count can live in totalSetData??

Reply:
They are not doing the same fetch